packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -46,6 +46,17 @@ const hatConfigSections = [
  * hats.  The decision about which hat styles should be available is up to
  * {@link VscodeEnabledHatStyles}
  */
+
+const SETTING_SECTION_HAT_SHAPES_DIR = "cursorless.private";
+const SETTING_NAME_HAT_SHAPES_DIR = "hatShapesDir";
+const hatShapesDirSettingId = `${SETTING_SECTION_HAT_SHAPES_DIR}.${SETTING_NAME_HAT_SHAPES_DIR}`;
```

Comment:
Made some constants so we don't forget to update one of them; in your version you'd forgotten to change one of the locations from `experimental` to `private`

Reply:
I saw that. Good catch